eth_keyfile/keyfile.py

```diff
@@ -83,7 +83,7 @@ def normalize_keys(keyfile_json):
 
 
 def _create_v3_keyfile_json(private_key, password, kdf, work_factor=None):
-    salt = Random.get_random_bytes(16)
+    salt = Random.get_random_bytes(32)
```

Comment:
I don't believe we can make this change as the default.  Can you look into how we could expose this at the public API level so that we can leave the 16-byte default in place but allow for users to supply a different value.